app/services/universal/promises.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+/**
+ * bluebird.props with native promises
+ * @param {object} props
+ * @returns {Promise}
+ */
+module.exports.props = function (props) {
+  return Promise.all(Object.keys(props).map(function (key) {
+    return Promise.resolve(props[key]).then(function (res) {
+      let one = {};
+
+      one[key] = res;
+      return one;
```

Comment:
I would suggest here a direct return, since you don't really need the `one` variable.

```
return Promise.resolve(props[key]).then(function (res) {
  return { key: res };
});
```

Reply:
Same with the one below

Reply:
If you ask me, I'd just use arrow functions 'cause I think it'll look more readable. But that's really up to you! Let me know what you think